src_kt/java/arcs/crdt/BUILD

```diff
@@ -8,5 +8,9 @@ kt_jvm_and_js_library(
     deps = [
         "//src_kt/java/arcs/common",
         "//src_kt/java/arcs/crdt/internal",
+        "//src_kt/java/arcs/crdt/entity",
+        "//src_kt/java/arcs/type",
+        # This may get unwieldy with avoiding circular deps as entity is implemented. We'll see.
```

Comment:
I'd say if you find cases where their are circular deps, and you need to refactor to break them,  in both Kotlin and TS, it will be beneficial, because one of the chief blockers of not being able to use ts_library to build the runtime in g3 is the ban on cyclical imports/deps.

Reply:
That sounds good, I'm taking things slowly - it might be nice to do a retrospective after this port with the aim of seeing what it would take to untangle the Typescript variant.